Conceptpower+Spring/src/main/java/edu/asu/conceptpower/app/lucene/impl/LuceneUtility.java

```diff
@@ -579,13 +609,14 @@ public void indexConcepts(String userName)
                 ConceptEntry entry = getConceptFromDocument(d);
                 concepts.add(entry);
             }
+            logger.debug("Number of concepts retrieved from lucene = " + concepts.size());
```

Comment:
remove

Reply:
isn't this logger required to log the number of concepts retrieved for the query? This was there from before.

Reply:
ya, that was never a really useful log message...

Reply:
Will remove the log message.